### ISSUE TITLE
Problem with IOS microphone permission

### ISSUE THREAD

**Issue by paolofru** [microsoft/cognitive-services-speech-sdk-js#583]
Hi, I saw another issue on this problem, it was about an older version of the bundle.js library that was causing problems with opening the microphone on ios.

This is my script:
       subscriptionKey = cred.cred_MICR.authorizationToken;
       serviceRegion = cred.cred_MICR.region;
       var phraseDiv = document.getElementById("phraseDiv");
       var cont_domanda = document.getElementById("cont_domanda");  

       speechConfig = SpeechSDK.SpeechConfig.fromAuthorizationToken(subscriptionKey, serviceRegion);
       speechConfig.speechRecognitionLanguage = "it-IT";

       var audioConfig  = SpeechSDK.AudioConfig.fromDefaultMicrophoneInput();
       recognizer = new SpeechSDK.SpeechRecognizer(speechConfig, audioConfig);

       recognizer.recognizeOnceAsync(
       function (result) {
           recognizer.close();
           recognizer = undefined;
           fromResponseQuestion(result.text)
         },
       function (err) {
           recognizer.close();
           recognizer = undefined;
           }
       );

The script works perfectly on desktop and android but doesn't work well on iphone. After activating the microphone and listening only once, the script closes the microphone and does not allow you to reopen it.

After isolating the code line by line, I realized that the main problem is in the line "recognizer.close();" , this code block the microphone and not allow to reopen it.

I tried in my script to update the microsoft.cognitiveservices.speech.sdk.bundle.js library to the latest version but the problem is not solved.

![IMG_3584](https://user-images.githubusercontent.com/1303727/194061343-566bb4f8-694a-4693-839c-d0f76ec17604.jpg)

Can you help me?

**Comment by glharper** [comment]
@paolofru Thank you for using JS Speech SDK, and writing this issue up.
Which browser are you using to reproduce this, and which version of iOS? I tested on iOS 16.0.2, Safari and Chrome with [this sample](https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/master/quickstart/javascript/browser/from-microphone/index.html) and found no issues repeatedly starting recognition. 

Could you test with that sample? Note the line:
```
 <script src="https://aka.ms/csspeech/jsbrowserpackageraw"></script>
 ```
 
 That line will ensure you're using the latest JS Speech SDK version.
